client/relay-chain-minimal-node/src/network.rs

```diff
@@ -116,6 +134,54 @@ pub(crate) fn build_collator_network(
 /// we provide a noop implementation.
 struct DummyChainSync;
 
+impl DummyChainSync {
```

Comment:
This does not need to be on the `DummyChainSync` no?

Reply:
It doesn't have to be but it's part of `ChainSync` in Substrate so I thought making it part of `DummyChainSync` would closest to what we have in Substrate

Reply:
On second thought, it might make more sense to have a separate method in substrate that exposes this. In case something changes we want the cumulus implementation to follow.

Reply:
I believe the vision is to make it more transparent so that you could instantiate some syncing engine implementation that would then install the necessary notification and request protocols to, e.g., `sc_network::Params` so that there would be no need for calling, e.g. `light_client_requests::generate_protocol_config()` (except for in the syncing implementation of course) but I'm not opposed to exposing this through Substrate as a temporary measure.

The fire I'm trying to put out right now though is that Polkadot CI is blocked by this PR. How do you want to proceed with this?

Reply:
Okay, I think its fine for now then, we could have a follow up.